lib/string_segmenter.rb

```diff
@@ -1,2 +1,10 @@
+require 'dictionary'
 def segment_string(str)
-end
\ No newline at end of file
+  words = []
+  str.downcase.delete('^a-z ')
+  if str.include dictionary
+    then words(//) << str
```

Comment:
(//) can format a space between words, but it has no place here, I deleted it.  What I want this to do is, for Ruby to go through the array, and if the dictionary words are in the dictionary array, to send the string into an empty array.  (I was hoping the // would help the array to seperate out the words and put spaces between them, but I dont know if thats possible).  Any way you cut it, it doesnt work, and gives me the error message of:Users/Meier/.rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- dictionary (LoadError)
	from /Users/Meier/.rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from string_segmenter.rb:1:in `<main>'